boards/ek-lm4f120xl/include/periph_conf.h

```diff
@@ -35,6 +36,7 @@ extern "C" {
 #define CLK16                           4
 #define CLK1                            5
 #define CLOCK_SOURCE                    CLK40
+#define CLOCK_CORECLOCK                 MHZ(80)
```

Comment:
I was unable to verify this, as only the errata sheet of the board but no longer the data sheet is available. However, the errata sheet seems to indicate that 80 MHz is the highest supported core frequency. So I just trust you on this.

Btw: The boards documentation is very much sub standard and the board itself is no longer being sold. I wonder if it makes sense to propose dropping support for it in the mailing list. (And if someone jumps and gives the board support some love instead of removing it, that would be an even better outcome.)

Reply:
EK-TM4C123G is compatible, I have one of those.
Ti must have given them away quite a lot at some point. But yea, the data sheet is not very great, Ti mostly expects you to use their DriverLib which is also available on the Chip ROM.